src/favoriteRestaurant.js

```diff
@@ -11,15 +11,30 @@ function hashRestaurantsCount(data) {
       restaurantCountHash[restaurant] += 1;
     });
```

Comment:
In modern JavaScript code `for..of` is generally preferred over `forEach()`

From the docs:

> forEach() expects a synchronous function — it does not wait for promises. Make sure you are aware of the implications while using promises (or async functions) as forEach callbacks.

> There is no way to stop or break a forEach() loop other than by throwing an exception. If you need such behavior, the forEach() method is the wrong tool.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach